src/client/components/TabPaneContent0/index.test.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import TabPaneContent0 from "./index";
+import { shallow } from "enzyme";
+
+describe("<TabPaneContent0 />", () => {
+  it("renders content pane", async () => {
+    const getTabLink = jest.fn();
```

Comment:
i guess we should have a unit test for this in tabbedcontainer, i'll make a ticket to flesh out tests (if there isn't already one)

Reply:
That sounds good, thanks!

Reply:
https://github.com/cagov/unemployment/issues/157